lib/1.4/dml-builtins.dml

```diff
@@ -1976,21 +1995,7 @@ template bank is (object, shown_desc, _qname) {
         // as it stands, DML prevents us from deducing the
         // bitorder directly so we rely on the XML 'guessing'
         method big_endian_bitorder() -> (bool) {
-            local attr_value_t args;
-            args = SIM_make_attr_list(
-                2,
-                SIM_make_attr_string(dev.classname),
-                SIM_make_attr_string(bank.name));
-
-            local attr_value_t ret;
-            ret = VT_call_python_module_function("register_view",
-                                                 "big_endian_bitorder",
-                                                 &args);
-            SIM_attr_free(&args);
-
-            local bool big_endian = SIM_attr_boolean(ret);
-            SIM_attr_free(&ret);
-            return big_endian;
+            return bank.be_bitorder;
```

Comment:
Could the changes to `big_endian_bitorder()` cause any potential breakage?

Reply:
No, for two reasons:
1. big_endian_bitorder only affects presentation, not simulation. When it returns true, some UIs will display bit numbers in the wrong order.
2. The autoparam is set to the same value as was written in the XML entry